Automattic/wp-calypso
# Conversation caterpillar: respect the has_avatar flag on authors

In the Reader API endpoints, we add a boolean flag called `has_avatar`. We set this to false whenever the avatar is missing, or additionally if it's a generic avatar (the "mystery man").

This PR updates the conversation caterpillar to omit authors with `has_avatar` set to `false`.

### To test

Head to http://calypso.localhost:3000/devdocs/blocks/conversation-caterpillar and make sure Boris has been omitted from the avatars (sorry Boris). I've set `has_avatar: false` on his fixture but left the avatar URL intact.
